### DISCUSSION TITLE
Add release tag to SCI release

### DISCUSSION THREAD

**Discussion by YaSuenag** [Green-Software-Foundation/sci#343] [Ideas]
GSF [announced](https://greensoftware.foundation/articles/green-software-foundation-expands-efforts-to-incentivize-decarbonization-in-t) SCI v1.0 has been published, and the article points this repository. However it does not point what revision is v1.0 .

Can we add release tag e.g. `v1.0` like general OSS projects? It is very helpful when we trac the difference from previous release in future.

**Comment by seanmcilroy29** [comment]
@YaSuenag - The SCI has been submitted to become an ISO standard, so the version release has been paused until this process has been concluded. During the submission process, ISO will more than likely provide feed back, which will subsequently require the WG to apply changes to the current SCI approved baseline. This iterative process may be required a few times, until the SCI Spec is approved by ISO.  Once (if) approved by ISO the GSF will add a version release to synchronise with ISO's version release tag. This will avoid any version confusion in the future.

**Comment by YaSuenag** [comment]
I know that, but [this article](https://greensoftware.foundation/articles/green-software-foundation-expands-efforts-to-incentivize-decarbonization-in-t) says as following:

> Measure: To measure emissions, organizations can now use the [Software Carbon Intensity (SCI) Specification v.1.0](https://grnsft.org/sci), a standardized protocol to calculate the rate of carbon emissions for a software system, for example carbon per minute for a streaming application.

The link of SCI v1.0 seems to point to HEAD of main branch of SCI repo, so v1.0 spec may vary depending on the timing of the reference.

For which revision was this article written? I think we should tag something to that revision.

**Comment by seanmcilroy29** [comment]
@YaSuenag - Thanks for pointing this out. I'll have a chat with the editor to have the updated correctly
